src/plugins/tbin/tbin/Map.cpp

```diff
@@ -0,0 +1,384 @@
+
+        std::string currTilesheet = "";
+        for ( int iy = 0; iy < ret.layerSize.y; ++iy )
+        {
+            int ix = 0;
+            while ( ix < ret.layerSize.x )
+            {
+                sf::Uint8 c = read< sf::Uint8 >( in );
+                switch ( c )
+                {
+                    case 'N':
+                        ix += read< sf::Int32 >( in );
+                        break;
+                    case 'S':
+                        ret.tiles[ ix + iy * ret.layerSize.x ] = std::move( readStaticTile( in, currTilesheet ) );
+                        ++ix;
+                        break;
+                    case 'A':
+                        ret.tiles[ ix + iy * ret.layerSize.x ] = std::move( readAnimatedTile( in ) );
+                        ++ix;
+                        break;
+                    case 'T':
+                        currTilesheet = read< std::string >( in );
+                        break;
+                    default:
+                        throw std::invalid_argument( QT_TR_NOOP( "Bad layer tile data" ) );
+                }
+            }
+        }
+
+        return std::move( ret );
+    }
+
+    void writeLayer( std::ostream& out, const Layer& layer )
+    {
+        write( out, layer.id );
+        write< sf::Uint8 >( out, layer.visible ? 1 : 0 );
+        write( out, layer.desc );
+        write( out, layer.layerSize );
+        write( out, layer.tileSize );
+        writeProperties( out, layer.props );
+
+        std::string currTilesheet = "";
+        for ( int iy = 0; iy < layer.layerSize.y; ++iy )
+        {
+            sf::Int32 nulls = 0;
+            for ( int ix = 0; ix < layer.layerSize.x; ++ix )
+            {
+                const Tile& tile = layer.tiles[ ix + iy * layer.layerSize.x ];
+
+                if ( tile.isNullTile() )
+                {
+                    ++nulls;
+                    continue;
+                }
+
+                if ( nulls > 0 )
+                {
+                    write< sf::Uint8 >( out, 'N' );
+                    write( out, nulls );
+                    nulls = 0;
+                }
+
+                if ( tile.tilesheet != currTilesheet )
+                {
+                    write< sf::Uint8 >( out, 'T' );
+                    write( out, tile.tilesheet );
+                    currTilesheet = tile.tilesheet;
+                }
+
+                if ( tile.animatedData.frames.size() == 0 )
+                {
+                    write< sf::Uint8 >( out, 'S' );
+                    writeStaticTile( out, tile );
+                }
+                else
+                {
+                    write< sf::Uint8 >( out, 'A' );
+                    writeAnimatedTile( out, tile );
+                }
+            }
+
+            if ( nulls > 0 )
+            {
+                write< sf::Uint8 >( out, 'N' );
+                write( out, nulls );
+            }
+        }
+    }
+
+    constexpr const char* MAGIC_1_0 = "tBIN10";
```

Comment:
I think VS 2013 is also not supporting member initialization at declaration, but in this case, we can just fix up this line and see if the build on AppVeyor finds anything else to complain about.